src/js/settings.js

```diff
@@ -1,135 +1,142 @@
+    formInputs: 'input, select',
+  },
+  menuProduct: {
+    clickable: '.product__header',
+    form: '.product__order',
+    priceElem: '.product__total-price .price',
+    imageWrapper: '.product__images',
+    amountWidget: '.widget-amount',
+    cartButton: '[href="#add-to-cart"]',
+  },
+  widgets: {
+    amount: {
+      input: 'input.amount',
+      linkDecrease: 'a[href="#less"]',
+      linkIncrease: 'a[href="#more"]',
     },
-    containerOf: {
-      menu: '#product-list',
-      cart: '#cart',
-      pages: '#pages',
-      booking: '.booking-wrapper',
-    },
-    all: {
-      menuProducts: '#product-list > .product',
-      menuProductsActive: '#product-list > .product.active',
-      formInputs: 'input, select',
-    },
-    menuProduct: {
-      clickable: '.product__header',
-      form: '.product__order',
-      priceElem: '.product__total-price .price',
-      imageWrapper: '.product__images',
-      amountWidget: '.widget-amount',
-      cartButton: '[href="#add-to-cart"]',
-    },
-    widgets: {
-      amount: {
-        input: 'input.amount', // CODE CHANGED
-        linkDecrease: 'a[href="#less"]',
-        linkIncrease: 'a[href="#more"]',
-      },
-      datePicker: {
-        wrapper: '.date-picker',
-        input: `input[name="date"]`,
-      },
-      hourPicker: {
-        wrapper: '.hour-picker',
-        input: 'input[type="range"]',
-        output: '.output',
-      },
-    },
-    booking: {
-      peopleAmount: '.people-amount',
-      hoursAmount: '.hours-amount',
-      tables: '.floor-plan .table',
-    },
-    nav: {
-      links: '.main-nav a',
-    },
-    
-    // CODE ADDED START
-    cart: {
-      productList: '.cart__order-summary',
-      toggleTrigger: '.cart__summary',
-      totalNumber: `.cart__total-number`,
-      totalPrice: '.cart__total-price strong, .cart__order-total .cart__order-price-sum strong',
-      subtotalPrice: '.cart__order-subtotal .cart__order-price-sum strong',
-      deliveryFee: '.cart__order-delivery .cart__order-price-sum strong',
-      form: '.cart__order',
-      formSubmit: '.cart__order [type="submit"]',
-      phone: '[name="phone"]',
-      address: '[name="address"]',
-    },
-    cartProduct: {
-      amountWidget: '.widget-amount',
-      price: '.cart__product-price',
-      edit: '[href="#edit"]',
-      remove: '[href="#remove"]',
-    },
-    // CODE ADDED END
-  };
-  
+    datePicker: {
+      wrapper: '.date-picker',
+      input: `input[name="date"]`,
+    },
+    hourPicker: {
+      wrapper: '.hour-picker',
+      input: 'input[type="range"]',
+      output: '.output',
+    },
+  },
+  booking: {
+    peopleAmount: '.people-amount',
+    hoursAmount: '.hours-amount',
+    tables: '.floor-plan .table',
+    tablesContainer: '.floor-plan',
+    starters: '[name="starter"]',
+    phone: '.order-confirmation [name="phone"]',
+    address: '.order-confirmation [name= "address"]',
+    tableSelected: '.table.selected',
+    form: '.booking-form',
```

Comment:
Tutaj chyba powinno być submit
```suggestion
    submit: '.booking-form',
```